pyproject.toml

```diff
@@ -24,6 +24,12 @@ opentelemetry-instrumentation-fastapi = "^0.47b0"
 opentelemetry-exporter-prometheus = "^0.47b0"
 aiocache = "^0.12.2"
 httpx = "^0.27.0"
+dynaconf = "^3.2.6"
+
+asyncpg = { version = "^0.29.0", optional = true }
+
+[tool.poetry.extras]
+postgresql = ["asyncpg"]
```

Comment:
I could not figure out how to reference the `tortoise-orm[asyncpg]` extra here.

Reply:
I think this is fine for now.